src/Sdk/DTLogging/Logging/ValueEncoders.cs

```diff
@@ -103,5 +108,26 @@ private static string Base64StringEscapeShift(String value, int shift)
 
             return result.ToString();
         }
+
+        private static String TrimBase64End(String value)
+        {
+            if (String.IsNullOrEmpty(value))
+            {
+                return String.Empty;
```

Comment:
will this cause all empty string become ***? will `value` ever be empty?

Reply:
We short circuit if the value is empty 
https://github.com/actions/runner/blob/master/src/Sdk/DTLogging/Logging/SecretMasker.cs#L111
https://github.com/actions/runner/blob/master/src/Sdk/DTLogging/Logging/SecretMasker.cs#L172